draft-ietf-lisp-pubsub.xml

```diff
@@ -342,15 +343,15 @@ Rec +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 
           <t>In the particular case of PubSub, cache poisoning via malicious Map-Notify messages is avoided by the use of nonce and the security association between the ITRs and the Map-Servers.</t>
 
-          <t>To prevent xTR-ID hijacking, it is RECOMMENDED to follow guidance from Section 9 of <xref target="RFC9301"></xref> to ensure integrity protection of Map-Request messages.</t>
+          <t>It is RECOMMENDED to follow guidance from the last paragraph of Section 9 of <xref target="RFC9301"></xref> to ensure integrity protection of Map-Request messages (e.g. to prevent xTR-ID hijacking).</t>
```

Comment:
```suggestion
          <t>It is RECOMMENDED to follow guidance from the last paragraph of Section 9 of <xref target="RFC9301"></xref> to ensure integrity protection of Map-Request messages (e.g., to prevent xTR-ID hijacking).</t>
```